stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/images/ImageSlidePageFragment.kt

```diff
@@ -14,12 +14,16 @@ public class ImageSlidePageFragment : Fragment() {
 
     private lateinit var binding: StreamUiItemImageGalleryBinding
 
+    private var imageClickListener: (() -> Unit)? = {}
```

Comment:
Since this is nullable, it could be `null` by default.

Reply:
Instead of that, doesn't allow null here, then we don't need to handle nullability

Reply:
We're clearing it in `onDestroyView` at the moment, that's why it's nullable - perhaps we don't need that though?

Reply:
Removed the nullable part.